frontend/webapp/components/setup/connection/create.connection.form/create.connection.form.tsx

```diff
@@ -179,14 +179,14 @@ export function CreateConnectionForm({
         {renderFields(fields, dynamicFields, handleDynamicFieldChange)}
       </DynamicFieldsWrapper>
       <CreateDestinationButtonWrapper>
-        <KeyvalButton disabled={isCreateButtonDisabled} onClick={onCreateClick}>
+        <KeyvalButton disabled={isCreateButtonDisabled} type='submit'>
```

Comment:
Thank you for your contribution, @Krishnakalani111. While the code seems to be functioning, there appears to be an issue that arises after form submission. Please double-check your code and resubmit it once the correction is made.